defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java

```diff
@@ -0,0 +1,112 @@
+package com.defect.tracker.controller;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.sql.Timestamp;
+
+import javax.mail.MessagingException;
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.defect.tracker.data.dto.EmployeeDto;
+import com.defect.tracker.data.entities.Employee;
+import com.defect.tracker.data.entities.VerificationToken;
+import com.defect.tracker.services.EmployeeService;
+import com.defect.tracker.services.VerificationService;
+import com.defect.tracker.util.EndpointURI;
+import com.defect.tracker.data.mapper.Mapper;
+import com.defect.tracker.data.response.ValidationFailureResponse;
+import com.defect.tracker.util.Constants;
+import com.defect.tracker.util.ValidationConstance;
+import com.defect.tracker.util.ValidationFailureStatusCodes;
+
+@RestController
+public class EmployeeController {
+	@Autowired
+	private EmployeeService employeeService;
+
+	@Autowired
+	private ValidationFailureStatusCodes validationFailureStatusCode;
+
+	@Autowired
+	private VerificationService verificationService;
+
+	@Autowired
+	private Mapper mapper;
+
+	@PostMapping(value = EndpointURI.EMPLOYEE)
+	public ResponseEntity<Object> addEmployee(@Valid @RequestPart("employee") String employee,
+			@RequestPart("file") MultipartFile file) throws IllegalStateException, IOException, MessagingException {
+		EmployeeDto employeeDto = employeeService.getJson(employee);
+
+		if (employeeService.isEmailAlreadyExist(employeeDto.getEmail())) {
+			return new ResponseEntity<>(new ValidationFailureResponse(ValidationConstance.EMPLOYEE_EMAIL_EXISTS,
+					validationFailureStatusCode.getEmpEmailAlreadyExists()), HttpStatus.BAD_REQUEST);
+		}
+
+		if ((int) file.getSize() > 1048576) {
+			return new ResponseEntity<>(
+					new ValidationFailureResponse(ValidationConstance.EMPLOYEE_PROFILE_SIZE_EXCEPTION,
+							validationFailureStatusCode.getEmpProfileSizeException()),
+					HttpStatus.BAD_REQUEST);
+		}
+
+		if (file.isEmpty()) {
+			return new ResponseEntity<>(new ValidationFailureResponse(ValidationConstance.EMPLOYEE_PROFILE_EMPTY,
+					validationFailureStatusCode.getEmpProfileIsEmpty()), HttpStatus.BAD_REQUEST);
+		}
+
+		if (!file.getContentType().equals("image/jpeg")) {
+			System.out.println(validationFailureStatusCode.getEmpProfileContenetTypeException());
```

Comment:
remove this print

Reply:
image validation  no need remove this validation